src/common/store/user.store.js

```diff
@@ -1,11 +1,10 @@
-import { create } from 'zustand';
-
 import { getEnvs } from 'common/functions';
+import { create } from 'zustand';
 
 export const useUserStore = create(set => ({
   subscriptionKey: '',
   setSubscriptionKey: key => set(() => ({ subscriptionKey: key })),
-  geography: Object.keys(getEnvs())[0],
+  geography: localStorage.getItem('geography') || Object.keys(getEnvs())[0],
```

Comment:
In the past we deliberately chose not to store any user data for compliance reasons. I guess it's not a big deal in this case (tho still may be worth double-checking), just wanted you to be aware of that

Reply:
Thanks for pointing it out. I will check with the team